static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.ts

```diff
@@ -467,21 +465,24 @@ const biddersSwitchedOn = (allBidders: PrebidBidder[]): PrebidBidder[] => {
 	return allBidders.filter((bidder) => isSwitchedOn(bidder));
 };
 
-const currentBidders = (slotSizes: HeaderBiddingSize[]): PrebidBidder[] => {
+const currentBidders = (
+	slotSizes: HeaderBiddingSize[],
+	pageTargeting: PageTargeting,
+): PrebidBidder[] => {
 	const biddersToCheck: Array<[boolean, PrebidBidder]> = [
 		[shouldIncludeCriteo(), criteoBidder],
 		[shouldIncludeSmart(), smartBidder],
-		[shouldIncludeSonobi(), sonobiBidder],
+		[shouldIncludeSonobi(), sonobiBidder(pageTargeting)],
 		[shouldIncludeTrustX(), trustXBidder],
 		[shouldIncludeTripleLift(), tripleLiftBidder],
-		[shouldIncludeAppNexus(), appNexusBidder],
+		[shouldIncludeAppNexus(), appNexusBidder(pageTargeting)],
 		[shouldIncludeImproveDigital(), improveDigitalBidder],
 		[shouldIncludeImproveDigitalSkin(), improveDigitalSkinBidder],
 		[shouldIncludeXaxis(), xaxisBidder],
 		[true, pubmaticBidder],
 		[shouldIncludeAdYouLike(slotSizes), adYouLikeBidder],
-		[shouldUseOzoneAdaptor(), ozoneClientSideBidder],
-		[shouldIncludeOpenx(), openxClientSideBidder],
+		[shouldUseOzoneAdaptor(), ozoneClientSideBidder(pageTargeting)],
+		[shouldIncludeOpenx(), openxClientSideBidder(pageTargeting)],
```

Comment:
I really like this pattern of passing down the pageTargeting value here, makes it much clearer from a glance what bidders rely on it.